fv3_cap.F90

```diff
@@ -1185,6 +1157,9 @@ subroutine ModelFinalize(gcomp, rc)
       enddo
     endif
 
+    call ESMF_VMBarrier(vm=vm, rc=rc)
```

Comment:
I don't think this Barrier() is needed, and should not be added.

Reply:
Removed in 48768632629a583b5b55ca0239493da655ec7a5a